app/main.py

```diff
@@ -14,4 +14,8 @@ def count_occurrences(phrase: str, letter: str) -> int:
     :param letter: letter to find occurrences of it
     :return: count occurrences of letter in phrase
     """
-    # write your code here
+    counter = 0
+    for char in phrase:
+        if char.upper() == letter.upper():
+            counter += 1
+    return counter
```

Comment:
try to write it in one line

Reply:
added "Solution v2". Should i do another pull request?